frontend/src/app/components/IncompatibleAddons/webpack.config.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.(css|scss)$/,
+        loaders: ["style-loader", "css-loader", "sass-loader"],
```

Comment:
The sass-loader docs gave something very different than this. I never would have figured this out...

Reply:
Ignore that... it didn't work and shouldn't have been there

Reply:
FWIW, these are the changes to the main webpack config, basically from the sass-loader docs: https://github.com/mozilla/testpilot/pull/2859/files#diff-11e9f7f953edc64ba14b0cc350ae7b9d

Reply:
Yeah that changeset looks really good. I think I can actually understand how it works!